src/components/Filters/FilterGroupContext.ts

```diff
@@ -1,5 +1,36 @@
-import { createContext, useContext } from 'react';
-import { UseCollapseOutput } from 'react-collapsed/dist/types';
+import { createContext, useContext, CSSProperties, ReactNode, TransitionEvent, MouseEvent } from 'react';
+
+
+declare type AriaBoolean = boolean | 'true' | 'false';
+
+export interface GetTogglePropsOutput {
```

Comment:
why are we redefining all the react-collapsed types? can't we get them from the library?

Reply:
on the latest version of react-collapse, they removed the /dist/types folder that had all the types laid out. so i had to redefine them to get the types.

Reply:
they don't export the types from anywhere anymore? I thought their release notes indicated they had added better types.

if they don't define them anywhere, then can we derive the types from `useCollapse` rather than hard-coding them ourselves? hard-coding the types could be problematic if they ever change them in the future, because we'd have to keep checking and remain in sync with the library